src/script/api/script_waypointlist.hpp

```diff
@@ -34,7 +34,7 @@ class ScriptWaypointList : public ScriptList {
 class ScriptWaypointList_Vehicle : public ScriptList {
 public:
 	/**
-	 * @param vehicle_id The vehicle to get the list of waypoints he has in its orders from.
+	 * @param vehicle_id The vehicle to get the list of waypoints from it has in its orders.
```

Comment:
Both old and new versions are confusing and ungrammatical.

"The vehicle from whose orders to get a list of visited waypoints" or something?

It would be useful to mention whether this is unique (in case a waypoint is in multiple orders) or sorted by the order sequence, but I don't know the answers... That bit is probably out of scope.